docs/content/node-operation/machine-existing-operator.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Machine Account for existing Node Operators
+sidebar_title: Machine Account for existing Node Operators
+description: Machine Account for existing Node Operators
+---
+
+If you are an existing `consensus` or `collection` node operator you will need a [Machine Account]() to participate in epochs. 
+This guide will walk you through creating a [Machine Account]() and getting it set up.
+
+## Downloading Bootstrap Utility
+
+<Callout type="warning">
+If you have downloaded the bootstrapping kit previously, ensure that you do this step again to get the latest copy of the bootstrapping kit since we have made some significant changes to it.
```

Comment:
```suggestion
If you have downloaded the bootstrapping kit previously, ensure that you do this step again to get the latest copy of the bootstrapping kit since there have been significant changes to it.
```